src/Controller/EnMarche/UserController.php

```diff
@@ -26,13 +28,30 @@ class UserController extends Controller
 
     /**
      * @Route("", name="app_user_profile")
-     * @Method("GET|POST")
+     * @Method("GET")
      */
     public function profileOverviewAction(): Response
     {
         return $this->render('user/my_account.html.twig');
     }
 
+    /**
+     * @Route("/mes-dons", name="app_user_profile_donation")
+     * @Method("GET")
+     */
+    public function profileDonationAction(): Response
```

Comment:
You should add an orderBy here, and you can simplify this controller : 

```
        $donations = $this->getDoctrine()->getRepository(Administrator::class)
            ->findBy(
                ['emailAddress' => $this->getUser()->getEmailAddress()],
                ['donatedAt' => 'ASC']
            )
        ;

        return $this->render('user/my_donation.html.twig', [
            'donations' => $donations
        ]);
```
But IMHO, you should make a repository method with this request logic.

Reply:
nice catch @thomasbisignani 
yeah this should be done in a DonationRepository with the query builder
```
//..
$qb->orderBy('donation.donatedAt', 'ASC');
//..
```

Reply:
done